tutorials/evoked/plot_eeg_erp.py

```diff
@@ -108,6 +110,59 @@
 evoked_custom.plot(titles=dict(eeg=title), time_unit='s')
 evoked_custom.plot_topomap(times=[0.1], size=3., title=title, time_unit='s')
 
+###############################################################################
+# Global field power (GFP)
+# ------------------------
+#
+# Global field power :footcite:`Lehmann1980,Lehmann1984,Murray2008` is,
+# generally speaking, a measure of agreement of the signals picked up by all
+# sensors across the entire scalp: if all sensors produce the same signal,
+# the GFP will be zero; if the signals differ, the GFP will be non-zero. GFP
+# peaks may be indicative of "interesting" brain activity, warranting further
+# investigation. Mathematically, the GFP is nothing but the population standard
+# deviation across all sensors, calculated separately for every time point.
+#
+# For visualization, you can simply use `evoked.plot() <mne.Evoked.plot>` and
+# add the GFP trace to the butterfly plot by passing ``gfp=True``. Let's also
+# use spatial coloring for the channel traces:
+
+evoked_car.plot(gfp=True, spatial_colors=True)
+
+###############################################################################
+# We used the average-referenced evoked signal here, the choice of a different
+# reference won't change the GFP:
+
+evoked_no_ref.plot(gfp=True, spatial_colors=True)
+
+###############################################################################
+# It is difficult to find out the absolute values of the GFP in the plots we
+# produced above. By passing ``gfp='only'``, the channel traces are removed,
+# and we can focus on the GFP exclusively. Although we don't plot any channel
+# traces, we still pass ``spatial_colors=True`` to get the GFP in a nice
+# black-and-gray color scheme – without this parameter, it would be
+# bright-green.
+
+evoked_car.plot(gfp='only', spatial_colors=True)
```

Comment:
```suggestion
# To plot the GFP by itself you can pass ``gfp='only'`` (this makes it easier
# to read off the GFP data values, because the scale is aligned):

evoked_car.plot(gfp='only')
```

[nitpick] nothing wrong with showing the green here IMO; it doesn't look that bad and it's informative to see it both ways.

More substantively, this makes me wonder why we don't add a second y scale on the right side of the axes that aligns to the zero value of the GFP trace (something for a separate PR).

Reply:
> More substantively, this makes me wonder why we don't add a second y scale on the right side of the axes that aligns to the zero value of the GFP trace (something for a separate PR).

I would love either that, or we create a 2nd axis for the GFP. Additional scale is probably much more space-preserving.